BUILD

```diff
@@ -11,3 +11,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+config_setting(
+    name = "clang",
+    flag_values = {"@bazel_tools//tools/cpp:compiler": "clang"},
```

Comment:
Just to confirm: Is this value being affected by setting the `CC` and `CXX` environment vars?

Reply:
This flag is set to the value of the `compiler` attribute of the `cc_toolchain`, which will be `"clang"` either if users use a hermetic clang toolchain or rely on the auto-detected host toolchain with `CC` set to `clang` on Linux (see https://github.com/bazelbuild/bazel/blob/f141e149ac3210e417b31aec21afed267cdbb092/tools/cpp/unix_cc_configure.bzl#L480). 

This currently doesn't detect `clang` correctly on macOS if Apple clang is used instead of the LLVM version of clang. If we want the FuzzedDataProvider fuzz test example to run there, we could always enable it on macOS (which will almost always use clang).

Reply:
Thanks for clarifying!